module/VuFindDevTools/tests/unit-tests/src/VuFindTest/Controller/DevtoolsControllerTest.php

```diff
@@ -70,7 +70,7 @@ public function testLanguageAction()
         $this->assertEquals('Zend\I18n\Translator\TextDomain', get_class($en['object']));
 
         // Did the @parent_ini macro get stripped correctly?
```

Comment:
This comment should no longer refer to `@parent_ini`... but maybe this whole assertion will go away when you eliminate `@fallback` as well. We should make sure there's a test for `@extend`, though!

Reply:
> This comment should no longer refer to `@parent_ini`... but maybe this whole assertion will go away when you eliminate `@fallback` as well.

Yep, this assertion can be removed as it will not fail in any case.

But moreover, I discovered the `VuFindDevTools` language components to need more attention in the light of this pull request: for example now the language helper detects `en-gb` to miss 1440 translations. Which is technically correct as neither the configured fallbacks nor any of the additionally registered language directories are considered.

I will think about this more thoroughly and especially if and how the insofar needed refactorization can be aligned with [our above mentioned efforts](https://github.com/vufind-org/vufind/pull/1232#discussion_r224052782) to develop a translation management user interface.

Reply:
@kese, good point regarding the dev tools. I would certainly be happy to spend some time bringing the tools up to date if you would like my assistance -- that would be a good opportunity for me to do some hands-on testing, etc. On the other hand, if the tools you are developing render the existing language tool obsolete, and if they are in a state that can be shared, I'm also open to completely replacing it as long as I still have a way of easily obtaining missing string lists for the translation team.

Reply:
I've added a TODO checkbox so we don't lose track of this issue before merging. There are a lot of simultaneous conversations going on in this PR, so please feel free to add additional TODO items if it helps keep track of status!